savoirfairelinux/num2words
# fix: TR language 'Num2Word_TR' object has no attribute 'str_to_number' error fixed

## Fixes # by...
Tufan Kaynak
### Changes proposed in this pull request:

* The code used an old import so updated

### Status


### How to verify this change

*Fill out this section so that a reviewer can know how to verify your change.*
num2words 1000 -l tr
should return
bin
which is the expected output

### Additional notes

*If applicable, explain the rationale behind your change.*
Received
xxxx 'Num2Word_TR' object has no attribute 'str_to_number'
error when executed
num2words xxxx -l tr
due to outdated import
